### ISSUE TITLE
prefixType not working like described in documentation

### ISSUE THREAD

**Issue by metapublic-gbr** [FriendsOfTYPO3/content-blocks#49]
In Typo3 112.4.7 and Content-Blocks 0.1.6, define 2 content-blocks:

```
name: ab/ce1
group: common
prefixFields: true
prefixType: vendor
fields:
  -
    identifier: select
    type: Select
```
```
name: ab/ce2
group: common
prefixFields: true
prefixType: vendor
fields:
  -
    identifier: select
    type: Select
```
flush caches in install-tool
run db-analyser.
result: 
```
Add fields to tables
    ALTER TABLE `tt_content` ADD `ab_ce1_select` VARCHAR(255) DEFAULT '' NOT NULL
    ALTER TABLE `tt_content` ADD `ab_ce2_select` VARCHAR(255) DEFAULT '' NOT NULL
```
expected result:
```
Add fields to tables
    ALTER TABLE `tt_content` ADD `ab_select` VARCHAR(255) DEFAULT '' NOT NULL
```

**Comment by nhovratov** [comment]
The `prefixType` feature is only available since Content Blocks 0.2.0

**Comment by nhovratov** [comment]
Closing this issue. Please update to the latest version to make use of this feature.
